ibtk/src/boundary/HierarchyGhostCellInterpolation.cpp

```diff
@@ -209,12 +197,12 @@ HierarchyGhostCellInterpolation::initializeOperatorState(
     VariableDatabase<NDIM>* var_db = VariableDatabase<NDIM>::getDatabase();
     bool registered_coarsen_op = false;
     d_coarsen_alg = new CoarsenAlgorithm<NDIM>();
-    for (unsigned int comp_idx = 0; comp_idx < d_transaction_comps.size(); ++comp_idx)
+    for (auto& d_transaction_comp : d_transaction_comps)
```

Comment:
This is nit-picky, but `d_XXX` variable names are reserved for non-static class member variables (with `s_XXX` being reserved for static class member variables).

Reply:
So, `d_transaction_comp` ---> `transaction_comp`.

Reply:
I think it should be easy to use a regex to convert `for (auto* d_` to `for (auto* ` and then clean up the resulting mangled variable names.

Reply:
I left it as `d_transaction_comp` since it is a reference to an element of a container variable (kind of like a member variable?), but I can go back and change them.